test/index_test.js

```diff
@@ -195,5 +195,69 @@ describe("Kinto", () => {
 
       expect(coll.db).to.be.an.instanceOf(MyAdapter);
     });
+
+    it("should set collection's plugins", () => {
+      const MyRemoteTransformer = class extends RemoteTransformer {};
+      const db = new Kinto();
+      const coll = db.collection("plop", {
+        remoteTransformers: [ new MyRemoteTransformer() ]
+      });
+
+      expect(coll._plugins.remoteTransformers).to.be.an.instanceOf(Array);
+      expect(coll._plugins.remoteTransformers.length).to.equal(1);
+      expect(coll._plugins.remoteTransformers[0])
+          .to.be.an.instanceOf(MyRemoteTransformer);
+    });
```

Comment:
I think we should try to avoid testing provate property getters, especially here where it's already tested by public API code using that specific piece of the impl.